pkg/minikube/audit/report_test.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
```

Comment:
add integration tes to test verison upgrade

Reply:
Added integration test